src/others/observer/lv_observer.h

```diff
@@ -271,6 +271,13 @@ void lv_observer_remove(lv_observer_t * observer);
  */
 void lv_subject_remove_all_obj(lv_subject_t * subject, lv_obj_t * obj);
 
+/**
+ * Remove the observers of an object from a subject
+ * @param obj       the object whose observers should be removed
+ * @param subject   the subject to remove the object from, or `NULL` to remove from all subjects
+ */
+void lv_obj_remove_from_subject(lv_obj_t * obj, lv_subject_t * subject);
```

Comment:
Maybe add a line to the doc comment to explain the difference between this and `lv_subject_remove_all_obj`, and/or add a line to the `lv_subject_remove_all_obj` doc comment mentioning this.